ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/poe.j2

```diff
@@ -0,0 +1,65 @@
+{# doc - poe #}
+{% set POE_CLASS_MAP = {0: "15.40", 1: "4.00", 2: "7.00", 3: "15.40", 4: "30.00", 5: "45.00", 6: "60.00", 7: "75.00", 8: "90.00"} %}
+{% set poe_global_defaults = namespace(reboot_action="power-off", interface_shutdown_action="maintain", lldp_negotiate=true) %}
+{% set ethernet_interfaces_poe = [] %}
+{% for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
+{%     if ethernet_interface.poe is arista.avd.defined %}
+{%         do ethernet_interfaces_poe.append(ethernet_interface) %}
+{%     endif %}
+{% endfor %}
+{% if (poe is arista.avd.defined) or (ethernet_interfaces_poe | length > 0) %}
+## Power Over Ethernet (PoE)
+
+### PoE Summary
+{%     if poe is arista.avd.defined %}
+
+#### PoE Global
+
+| Reboot Action | Shutdown Action | LLDP Negotiation |
+| ------------------- | -------------------- | ----------------------|
+{# Get global poe settings for defaults #}
+{%         set poe_global_defaults.reboot_action = poe.reboot.action | arista.avd.default(poe_global_defaults.reboot_action) %}
+{%         set poe_global_defaults.interface_shutdown_action = poe.interface_shutdown.action | arista.avd.default(poe_global_defaults.interface_shutdown_action) %}
+{#         get lldp transmit power via mdi #}
+{%         if lldp.tlvs is arista.avd.defined %}
+{%             for tlv in lldp.tlvs if tlv.name == 'power-via-mdi' %}
+{%                 set poe_global_defaults.lldp_negotiate = tlv.transmit %}
+{%             endfor %}
+{%         endif %}
+| {{ poe_global_defaults.reboot_action }} | {{ poe_global_defaults.interface_shutdown_action }} | {{ poe_global_defaults.lldp_negotiate }} |
+{%     endif %}
```

Comment:
I am now wondering if we should not render this all the time given that your are providing default values for the three categories and in that case I would change line 3 to be

```
{% set poe_global_defaults = namespace(reboot_action="power-off (default)", interface_shutdown_action="maintain (default)", lldp_negotiate="True (default)") %}
```

Reply:
Yea that makes total sense.  Good thinking.

Reply:
But actually, how do we account for non-poe platforms now?  Unless, there's a way to check if a particular device rendering cli is poe-capable, I think it's best to not include the POE global documentation.

Reply:
you also make a good point - actually I agree with you and lets include it only when there is actually something to configure. Thanks for your latest change.
@ClausHolbechArista what do you think of the current status?

Reply:
In `eos_cli_config_gen` we try to avoid documenting EOS defaults. Otherwise we would have a lot of "stale" documentation. I suggest to only document what the user has defined. Never interpret EOS defaults at all, unless it is absolutely required for either valid configuration or documentation clarity. We normally just set `-` as the value, if it is unset by the user.

Reply:
In that case, the only default I think that should be defined is poe `enabled` or `disabled`